OsmAnd-java/src/net/osmand/binary/BinaryInspector.java

```diff
@@ -1068,18 +1067,7 @@ private void printPOIDetailInfo(VerboseInfo verbose, BinaryMapIndexReader index,
 				MapUtils.get31TileNumberY(verbose.lattop),
 				MapUtils.get31TileNumberY(verbose.latbottom),
 				verbose.getZoom(),
-				new SearchPoiTypeFilter() {
-					@Override
-					public boolean accept(PoiCategory type, String subcategory) {
-						return true;
-					}
-
-					@Override
-					public boolean isEmpty() {
-						return false;
-					}
-
-				},
+				BinaryMapIndexReader.EMPTY_SEARCH_POI_TYPE_FILTER,
```

Comment:
Name of the filter is wrong, it is not EMPTY_SEARCH_POI_TYPE_FILTER, possible name ACCEPT_ALL_POI_TYPE_FILTER